create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -52,6 +53,13 @@
     private static final String SCOPUS_XML_0000469852 = "2-s2.0-0000469852.xml";
     private static final String SCP_ID_IN_0000469852 = "0000469852";
     private static final String DOI_IN_0000469852 = "10.1017/S0960428600000743";
+    private static final String SCOPUS_XML_0000833530 = "2-s2.0-0000833530.xml";
+    private static final String XML_ENCODING_DECLARATION = "<?xml version=\"1.0\" encoding=\"UTF-8\" "
+                                                           + "standalone=\"yes\"?>";
+    private static final String HARDCODED_EXPECTED_TITLE_IN_0000833530 = "Measurement of A\n"
+                                                                         + "    <sup>bb</sup>\n"
+                                                                         + "    <inf>FB</inf> in hadronic Z decays "
+                                                                         + "using a jet charge technique</titletextTp>";
```

Comment:
There alot more information inside the xml output. Are we intrested in knowing what version of the xjd files are at compile time? I don't thinks so, so I didn't include it.

Reply:
We are probably interested in the namespaces

Reply:
Fixed